src/VisualStudio/Core/Test.Next/project.json

```diff
@@ -2,18 +2,8 @@
   "dependencies": {
     "Newtonsoft.Json": "8.0.3",
     "Moq": "4.2.1402.2112",
```

Comment:
tagging @jasonmalinowski 

That's pretty awful :)

So, we have multiple versions of the same DLL being written to the same location.  But we need the VS15 versions to win?  Is that it?   If so, that sounds like a pretty significant issue.  We should not define out build graph such that multiple projects end up writing different values to the same output path.

Jason, do you have advice on how we should best fix this?